login.html

```diff
@@ -4,6 +4,91 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>판다마켓 로그인</title>
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="images/favicon-16x16.png"
+    />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
   </head>
-  <body></body>
+  <body>
+    <main>
+      <div class="form-container">
+        <form class="form" id="loginForm" method="POST">
+          <div class="form-logo">
+            <a href="/login.html" aria-label="새로고침">
+              <img src="images/logo-396.png" alt="판다마켓 로고" width="396" />
+            </a>
+          </div>
+          <div class="form-contents">
+            <div class="form-control">
+              <label class="form-label" for="userEmail">이메일</label>
+              <input
+                class="form-input"
+                type="email"
+                id="userEmail"
+                name="userEmail"
+                autocomplete="email"
+                placeholder="이메일"
+                required
+              />
+            </div>
+            <div class="form-control">
+              <label class="form-label" for="userPassword">비밀번호</label>
+              <div class="visible-wrap">
+                <input
+                  class="form-input"
+                  type="password"
+                  id="userPassword"
+                  name="userPassword"
+                  autocomplete="current-password"
+                  placeholder="비밀번호"
+                  required
+                />
+                <button
+                  type="button"
+                  class="btn-password-visible off"
+                  title="패스워드 표시/숨김"
+                ></button>
+              </div>
+            </div>
+            <button class="btn-lg btn-primary">로그인</button>
```

Comment:
제출 버튼에는 type="submit" 속성을 추가해주면 해당 버튼을 클릭 시 폼 제출 동작이 발생한다는것을 명시적으로 알려줄수있어요! 

```suggestion
            <button type="submit" class="btn-lg btn-primary">로그인</button>
```